cohort-calls/README.md

```diff
@@ -0,0 +1,31 @@
+# README file: cohort calls in OLS-2
+
+During the 16-week long program:
+
+-   Mentors and mentees meet every 2 weeks [for a 30 minutes call](https://openlifesci.org/ols-2#mentor-mentee-calls)
+-   Mentees participate every ~2 weeks to [90-minutes cohort calls](https://openlifesci.org/ols-2#cohort-calls) during which the program leaders introduce new topics and resources, facilitate break-out discussions, and invite expert from the field to give talks
+-   Mentees can meet together for coworking sessions every 2 weeks [for a 30 minutes call](https://openlifesci.org/ols-2#coworking-calls)
+-   Mentors take part to [4 mentoring calls](https://openlifesci.org/ols-2#mentors-calls)
+
+*Below we have listed details for the cohort calls and coworking calls, along with the link of shared HackMD document that you will use for shared notes:*
```

Comment:
We don't need this file, I was only working on it so that I could create the HackMD links without forgetting about the date and week. It's totally fine to delete the table and redirect it to the schedule.